Exec/science/celldet/inputs.celldet

```diff
@@ -71,19 +71,23 @@ amr.derive_plot_vars = ALL
 
 #PROBIN FILENAME
 
-problem.T_l = 4.e9
-problem.T_r = 5.e7
+problem.T_l = 4.423e9
+problem.T_r = 2.e8
```

Comment:
you renamed these to T_ash and T_fuel, and similarly rho_ash and rho_fuel, so you need to update the inputs